modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -11,9 +11,14 @@ The console is only available after the {app-security-service} Service has been
 
 .Overview of the Mobile Security Console
 
-The {app-security-service} Console allows you to monitor applications, their respective versions, and disable versions of those applications.
+The {app-security-service} Console allows you to monitor mobile apps, deployed versions, and disable versions of those mobile apps.
 
-Opening the console lists all applications. Click on an application to view version details for that application.
+Opening the console lists all applications. Click on an application to view deployed versions of that mobile app.
+
+A deployed version is registered with Mobile Security after:
```

Comment:
Should this say "registered with the Mobile Security Service". I don't believe it's ever referred to as just "Mobile Security".